subtrack.Tests/SubscriptionCalculatorTests/GetNextPaymentDateTests.cs

```diff
@@ -17,10 +17,23 @@ public GetNextPaymentDateTests()
     }
 
     [Fact]
-    public void GetNextPaymentDate_ReturnsCorrectNextPaymentDate()
+    public void GetNextPaymentDate_ReturnsCorrectNextPaymentDateWeekly()
     {
         // Arrange
-        var subscription = new Subscription { LastPayment = new DateTime(2023, 6, 2) , FirstPaymentDay = new DateTime(2023, 6, 2).Day};
+        var subscription = new Subscription { LastPayment = new DateTime(2023, 6, 2) , FirstPaymentDay = new DateTime(2023, 6, 2).Day, BillingInterval = 1, BillingOccurrence=BillingOccurrence.Week};
```

Comment:
```suggestion
        var subscription = new Subscription { LastPayment = new DateTime(2023, 6, 2) , FirstPaymentDay = 2, BillingInterval = 1, BillingOccurrence=BillingOccurrence.Week};
```

Personal opinon: Can do the same for firstpaymentday on other tests, its still hardcoded but faster to read